modules/ROOT/pages/promote-app-prod-env-design-center.adoc

```diff
@@ -1,34 +1,30 @@
-= Deploy a Mule Application (Design Center)
+= Deploy a Mule Application (Flow Designer)
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: mozart, deploy, environments
 
-When a Mule application that you've developed in the Design environment is ready to go live, you can deploy it to another environment. You must select an target environment, such as Production, that has been set up with vCores for deployment. You deploy an app to a specific environment for a number of reasons:
+When a Mule application that you've developed in the Design environment is ready to go live, you can deploy it to another environment. The target environment must have enough vCores to accommodate the application; otherwise, deployment will fail.
```

Comment:
It seems like you usually say Mule app (and I think that's preferred in SF style), so for consistency, you may want to S&R Mule application --> Mule app. Very minor and not necessary.